---
layout: page
title: Variant Dashboard Info
subtitle: About the Alaska Coronavirus Variant Dashboard
---
## Who created the dashboard?
This dashboard was created as part of the *Alaska SARS-CoV-2 Sequencing Consortium* which is an partnership between the University of Alaska, Alaska State Virology Lab, Department of Health and Social Services, and other state agencies with the aim to increase genomic surveillance of variants of concern (VOC) and variants of interest (VOI) in the state of Alaska. Also check out the weekly Alaska COVID-19 Genomic Surveillance [Situation Report](http://dhss.alaska.gov/covid19/HumanCoV/AKSeqCon_GenomicSituationReport.pdf) here.

## What is the significance of the variants dashboard?
Coronavirus variants have the potential to spread more rapidly, cause more severe disease, evade diagnostic detection, or reduce vaccine efficacy.  Timely and accurate surveillance of SARS-CoV-2 variants is key to informing public health policy and thus efforts to control the pandemic in Alaska.

## Where does the data come from?
On this dashboard you can find updates on a weekly basis on variants circulating in the state of Alaska based on data found in [GISAID](https://www.gisaid.org/).  Lineages are assigned using [Pangolin](https://cov-lineages.org/resources/pangolin.html).  To see how proportions of lineages in Alaska compare to national levels go to the [CDC's Variant Proportions page](https://covid.cdc.gov/covid-data-tracker/#variant-proportions).

## How to interpret SARS-CoV-2 Variant Dashboard?
Data used for this dashboard relies on shared virus sequences from the [GISAID Initiative](https://www.gisaid.org/), thus we rely on the accuracy of the data deposited there. Sequences are a sample of the total number of Alaska confirmed SARS-CoV-2 cases.  Although sequencing efforts aim to capture as many samples as possible, samples are often biased as a result of priority sampling for a single cluster in an outbreak, unique cases such as vaccine breakthroughs, cases imported through travel, etc. Moreover, there is a lag (often weeks) for all sequencing data between when the sample was collected from the patient, processed, sequenced, and released to the public in the form of the dashboard.  Thus data shown here contains few recent samples.
 

# Contact Us!
The *Variants Dashboard* is maintained by:
  - Tracie J. Haan, MS
    - email: tjhaan@alaska.edu
    - website: [tjhaan.github.io](https://tjhaan.github.io/)
  - Devin M. Drown, PhD
    - email: dmdrown@alaska.edu
    - UAF website: [Faculty Information](https://www.uaf.edu/bw/about/faculty.php?who=Drown_Devin)
    
#### This work is supported by Alaska INBRE, an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Healthunder grant number 2P20GM103395
